src/ripple/app/paths/cursor/DeliverNodeReverse.cpp

```diff
@@ -238,7 +244,8 @@ TER PathCursor::deliverNodeReverse (
             resultCode = increment(-1).deliverNodeReverse(
                 node().offerOwnerAccount_,
                 saInPassReq,
-                saInPassAct);
+                saInPassAct,
+                /*restart dir*/false);
```

Comment:
OK, with further reading I got it.  But now I think we can do a little better.

Let me first explain it.

When called from the top, `restartDir` is always `true` - because you only do the `restartDir` code the very first time and not on any subsequent calls.

But that fact shouldn't be exposed to callers.  And there's a neater way, gimme 5 mins...